hack/test.sh

```diff
@@ -25,8 +23,8 @@ if [ "$OPENSHIFT_CI" == "true" ] && [ -n "$ARTIFACT_DIR" ] && [ -d "$ARTIFACT_DI
 fi
 
 # Print the command we are going to run as Make would.
-echo ${GINKGO} ${GINKGO_ARGS} ${TEST_DIRS}
-${GINKGO} ${GINKGO_ARGS} ${TEST_DIRS}
+echo ${GINKGO} ${GINKGO_ARGS} ${GINKGO_EXTRA_ARGS} "<omitted>"
+${GINKGO} ${GINKGO_ARGS} ${GINKGO_EXTRA_ARGS} ./pkg/... ./cmd/...
```

Comment:
On this one, we are probably ok with just `./...`
```suggestion
${GINKGO} ${GINKGO_ARGS} ${GINKGO_EXTRA_ARGS} ./...
```